src/module/actor/familiar/document.ts

```diff
@@ -146,23 +147,12 @@ class FamiliarPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
 
         // Armor Class
         if (master) {
-            const source = master.system.attributes.ac.modifiers.filter(
-                (modifier) => !["status", "circumstance"].includes(modifier.type)
-            );
-            const base = 10 + new StatisticModifier("base", source).totalModifier;
-            const modifiers = extractModifiers(synthetics, ["ac", "dex-based", "all"]);
-            const stat = mergeObject(new StatisticModifier("ac", modifiers), systemData.attributes.ac, {
-                overwrite: false,
-            });
-            stat.value = base + stat.totalModifier;
-            stat.breakdown = [game.i18n.format("PF2E.MasterArmorClass", { base })]
-                .concat(
-                    stat.modifiers
-                        .filter((m) => m.enabled)
-                        .map((m) => `${m.label} ${m.modifier < 0 ? "" : "+"}${m.modifier}`)
-                )
-                .join(", ");
-            systemData.attributes.ac = stat;
+            const masterModifiers = master.armorClass.modifiers
+                .filter((m) => !["status", "circumstance"].includes(m.type))
+                .map((m) => m.clone());
+            const statistic = new ArmorStatistic(this, { modifiers: masterModifiers });
```

Comment:
If I recall, we can't pass the master modifiers directly, we need to get the result of said modifiers because any predication that passed for the master might no longer apply when evaluated from the perspective of the familiar.